tests/test_dependencies.py

```diff
@@ -508,7 +508,7 @@ def assert_dependencies(content: str):
 
         assert_dependencies(rendered1)
         self.assertEqual(
-            rendered1.count('Variable: <strong data-djc-id-a1bc41="" data-djc-id-a1bc42="">value</strong>'),
+            rendered1.count('Variable: <strong data-djc-id-a1bc42="" data-djc-id-a1bc41="">value</strong>'),
```

Comment:
When using `{% provide %}`, we also generate a unique ID for the provided data, hence the changes to the IDs in the tests